cmake/std/atdm/sems-rhel7/environment.sh

```diff
@@ -160,6 +160,20 @@ elif [[ "$ATDM_CONFIG_COMPILER" == "INTEL-17.0.1" ]] ; then
   if [[ "${ATDM_CONFIG_LM_LICENSE_FILE_OVERRIDE}" != "" ]] ; then
     export LM_LICENSE_FILE=${ATDM_CONFIG_LM_LICENSE_FILE_OVERRIDE}
   fi
+elif [[ "$ATDM_CONFIG_COMPILER" == "INTEL-18.0.5" ]] ; then
```

Comment:
Okay, so just adding a new compiler intel-18.0.5, not disturbing any of the existing compilers.